package.json

```diff
@@ -81,6 +81,8 @@
   "dependencies": {
     "@humanwhocodes/gitignore-to-minimatch": "^1.0.2",
     "@stylistic/eslint-plugin": "^2.9.0",
+    "eslint-import-resolver-typescript": "^3.6.3",
```

Comment:
We probably should import this ourselves, to ensure it can be resolved correctly.

But we don’t need to export it like we do with plugins, right?

Is it strictly necessary for TS-support? Or only for some features?

Reply:
As far as I known, it's an `eslint-plugin-import-x` peer deps used by it internally only with TS. 
I do not have better idea right now than including it in neostandard deps.

Reply:
If the plugin is hoisted but the resolver isn’t, then the plugin will not resolve our version but the one that prevented ours from being hoisted I believe.

If we can’t import it and provide it as an option to the plugin, then I think we have to make it a peer dependency for neostandard as well

One option is to skip this plugin for TS files until we can provide it as an option, and open a new issue for that in the plugin

Reply:
The plugin documentation says to install it for TS, but it's not a proper peer deps in package.json, so it's a bug. 
If the maintainers are reactive, the fix is easy. Peer deps are transitive, right?

Reply:
I don’t remember how they behave on newer npm versions, but if we add a module with a peer dependency, then we should be the one that ensures it’s fulfilled

They should change how the resolvers are defined in flat configs, it shouldn’t be a string: https://github.com/un-ts/eslint-plugin-import-x/blob/0bac033b5ff15f46cfb98760be24b3f33436e4a7/src/config/flat/typescript.ts#L27

Reply:
Until then we could maybe use `require.resolve()` to resolve an absolute path and use when defining the resolver